Week1/homework/js-excercises/e5-round a number.js

```diff
@@ -6,9 +6,18 @@ console.log(z);
 let a= Math.floor(z);
 console.log(a);
 
-let highest=[]
-if (z > a) {
-    return  console.log(highest=z);
-} else{
-    return  console.log(highest=a)
-};
+// let highest=[]
+// if (z > a) {
+//     return  console.log(highest=z);
+// } else{
+//     return  console.log(highest=a)
+// };
+
+let highest;
+if (z>a) {
+    highest=z
+}else {
+    highest=a
+}
+
+console.log(highest)
```

Comment:
Ah, nice! You can also use [Math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)

And don't forget the `;` at the end of the lines,.